content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -571,7 +571,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   kubelets on Pod log requests.
 - `AnyVolumeDataSource`: Enable use of any custom resource as the `DataSource` of a
   {{< glossary_tooltip text="PVC" term_id="persistent-volume-claim" >}}.
-- `AppArmor`: Enable AppArmor based mandatory access control on Linux nodes when using Docker.
+- `AppArmor`: Enable AppArmor based mandatory access control on Linux nodes when using container runtime.
```

Comment:
```suggestion
- `AppArmor`: Enable use of AppArmor mandatory access control for Pods running on Linux nodes.
```
perhaps?

Does this feature gate require Docker Engine? I assume that it doesn't; if it does, we need to address that.

Reply:
To my knowledge, AppArmor is concerned mainly with LXD, don't have much knowledge of docker engine requirements.

Reply:
AppArmor feature should work with other container runtimes other than docker, according to `Container runtime supports AppArmor` point in the [Restrict a Container's Access to Resources with AppArmor](https://kubernetes.io/docs/tutorials/security/apparmor/) documentation.

Reply:
> `AppArmor`: Enable AppArmor based mandatory access control on Linux nodes when using container runtime.

The wording as it stands doesn't make sense, because nodes _always_ use a container runtime (unless they are misconfigured, which doesn't really count).

Reply:
How about accepting the suggestion I made earlier @SubhasmitaSw ?

Reply:
The suggestions look great to me. 

> How about accepting the suggestion I made earlier @SubhasmitaSw ?

Reply:
> > `AppArmor`: Enable AppArmor based mandatory access control on Linux nodes when using container runtime.
> 
> The wording as it stands doesn't make sense, because nodes _always_ use a container runtime (unless they are misconfigured, which doesn't really count).

then something must be done to atone the dilemma on the documentation part?